spec/support/data_api/data_api_spec_helper.rb

```diff
@@ -10,7 +10,7 @@ def get_api_request(url, token:, options: {})
   end
 
   def tad_api_token
-    @tad_api_token ||= ServiceAPIUser.tad_user.create_magic_link_token!
+    @tad_api_token = ServiceAPIUser.tad_user.create_magic_link_token!
```

Comment:
@steventux I stole your idea for this fix as well and seemed to resolve the flakey here